Readme.MD

```diff
@@ -1212,3 +1213,70 @@ debugTouchArea        | bool     | Yes      | false                     | Set th
 animateTransitions    | bool     | Yes      | false                     | Set to true if you want to use the default 'spring' animation
 animationType         | string   | Yes      | 'timing'                  | Set to 'spring' or 'timing' to use one of those two types of animations with the default [animation properties](https://facebook.github.io/react-native/docs/animations.html).
 animationConfig       | object   | Yes      | undefined                 | Used to configure the animation parameters.  These are the same parameters in the [Animated library](https://facebook.github.io/react-native/docs/animations.html).
+
+## Tile Component
+
+A component with full size image and with text either inside the image or under the image along with customizable caption
+
+> This component was inspired from [Shoutem UI](https://github.com/shoutem/ui) by [Shoutem](https://github.com/shoutem). Check out [Shoutem](http://shoutem.github.io/) if you haven't already!
+
+#### Featured Tile
+
+```
+<Tile
+   imageSrc={{require: ('./img/path')}}
+   title="Lorem ipsum dolor sit amet, consectetur adipisicing elit. Dolores dolore exercitationem"
+   featured
+   caption="Some Caption Text"
+/>
+```
+
+![screen shot 2017-01-15 at 9 50 15 pm](https://cloud.githubusercontent.com/assets/6476108/21969491/beea4630-db6c-11e6-8913-7cc8813e35d6.png)
+
```

Comment:
Also here, can you switch the order of `title/code snippet /image` to `title/image/code snippet`?
Most of the other elements are organized in this manner.